_topic_maps/_topic_map_osd.yml

```diff
@@ -51,7 +51,7 @@ Name: Getting started
 Dir: osd_quickstart
 Distros: openshift-dedicated
 Topics:
-- Name: Quickstart for OpenShift Dedicated
+- Name: Getting started with OpenShift Dedicated
   File: osd-quickstart
 ---
 Name: Red Hat OpenShift Cluster Manager (OCM)
```

Comment:
I can see the "OCM" acronym in here a few times but I am thinking Eric's PR will be fixing this. Maybe we can double-check after?

Reply:
@dayleparker thank you for your detailed review!

In this PR, I have referred to OpenShift Cluster Manager by using the `{OCM}` attribute. From a more general perspective, I think that we should remove the "OCM" instances from the docs in a separate PR.